caffe2/onnx/onnx_exporter.cc

```diff
@@ -337,6 +343,53 @@ ConvertedResult OnnxExporter::CommonCaffe2OpToOnnxNodes(
   return result;
 }
 
+ConvertedResult OnnxExporter::CreateBinaryElementwiseOpNodes(
+    const caffe2::OperatorDef& def,
+    const std::unordered_map<std::string, caffe2::TensorShape>& shapes) {
+  int64_t end_dim = 0;
+  const auto& x = def.input(0);
+  const auto& y = def.input(1);
+  const auto& z = def.output(0);
+  const auto& x_shape = shapes.at(x);
+  const auto& y_shape = shapes.at(y);
+  const auto& z_shape = shapes.at(z);
+  for (const auto& arg: def.arg()) {
+    if (arg.name() == "axis") {
+      int64_t axis = arg.i();
+      if (x_shape.dims().size() - axis != y_shape.dims().size()) {
+        end_dim = y_shape.dims().size() - 1 - axis + x_shape.dims().size();
+      }
+      break;
+    }
+  }
+
+  if (end_dim > 0) {
+    ConvertedResult result;
+    auto& nodes = result.first;
+
+    std::vector<int64_t> unsqueezed_dims(y_shape.dims().begin(), y_shape.dims().end());
```

Comment:
`unsqueezed_dims` doesn't seem to be used?

Reply:
Yeah, I was confused by const_nodes, turn out it's useless. Something I need to clean up. Thanks, good catch.